docs/release_process.rst

```diff
@@ -0,0 +1,117 @@
+
+- Update the ``VERSION`` number in ``lore/settings.py`` following Semantic
+  numbering practice.
+- ``git-release-notes`` takes a range of git references to determine which
+  commits to include in the release.  In the example given it is all commits
+  from the last release to what is in ``master``.
+- Generate the release notes in restructuredtext format:
+
+.. code-block:: bash
+
+  git-release-notes v0.9.0..master util/release_notes_rst.ejs
+
+- Update ``release.rst`` with this version's release notes (the file
+  generated by running ``release_notes_rst.ejs``).
+
+4. Create Pull Requests
+-----------------------
+
+Push the rc branch and make three PRs:
+  - one to ``master`` called ``Release 0.10.0``
+  - one to ``release`` called ``rc/0.10.0`` containing a checklist of commits
+  - one to ``release-candidate`` called ``rc/0.10.0`` which should be merged
+    promptly to kick of the build
+
+Run ``git-release-notes`` again to create the PR description.
+
+.. code-block:: bash
+
+  git-release-notes v0.9.0..master util/release_notes.ejs
+
+and paste the Markdown output as the description of the PR to the release
+branch ``rc/0.10.0`` (This is the one with the checkboxes.)
+
+Tag the one to master to be reviewed ``Release 0.10.0``
+
+.. code-block:: bash
+
+  ``git tag Release 0.10.0``
+
+Tell developers to test their changes on the LORE release candidate
+server, http://lore-release.herokuapp.com
+and then bug team members until all the check boxes are checked.
+
+5. Hand Off Release to Dev-Ops
+------------------------------
+Lift the freeze after running migrations and then merging both the
+branches "finishing" the release
+
+When all the boxes are checked finish the release
+
+``git flow release finish 0.10.0``
+
+6. Send release notification to LORE mailing list
+--------------------------------------------------
+The email address of the LORE mailing list we use for release notification
+is odl-lore-announce@mit.edu It is a mailman list located here:
+https://mailman.mit.edu:444/mailman/admin/odl-lore-announce/general
+The list is set to hold all emails for review by a list moderator who
+are notified by email when an email is received.
+
+Release Process Roles
+=====================
+
+the important piece for the dev-ops person is migrations
+so the devops person should actually do the merge to release so they can run
+migrations before they do so which means we are almost exactly following the
+edX release strategy, where there is a release master and "dev-ops buddy".
+
+Peter's notes
+=============
+
+Don't hesitate to merge to the ``release-candidate`` branch, because
+folks will not wait for it to show up to start their verification tests.
+In retrospect, I probably should have done this *before* creating the PR
+to the ``release`` branch with all the checkboxes. The one to the
+``release-candidate`` branch should be merged immediately to kick
+of the build.
+
+``git flow release finish 0.10.0``
```

Comment:
We should add steps throughout about confirming that the builds were successful.